### ISSUE TITLE
Deploying local sql db to Azure 

### ISSUE THREAD

**Issue by AnthonySalmeron** [microsoft/azuredatastudio#14947]
**Is your feature request related to a problem? Please describe.**
I'd like a way to deploy a local SQL database to Azure seamlessly from the UI.
**Describe the solution or feature you'd like**
In the same way the SSMS allows you to right click on a local database and deploy it the Azure, I'd like a similar feature for Azure Data Studio. This would be really helpful for people using non-Windows machines.

**Comment by Charles-Gagnon** [comment]
We will triage this for a general improvement to the migration/import flow - but have you looked at using the Dacpac import feature?

https://docs.microsoft.com/en-us/sql/azure-data-studio/extensions/sql-server-dacpac-extension?view=sql-server-ver15

You would extract the Dacpac from the source DB and then deploy it to an Azure instance using the wizard.

**Comment by AnthonySalmeron** [comment]
Yes, that's what I eventually ended up using for the data migration I was performing. Thanks for the follow up.
